edx_proctoring/tests/__init__.py

```diff
@@ -20,6 +21,18 @@ def setup_test_backends():
     config.backends['mock'] = MockProctoringBackendProvider()
 
 
+@contextlib.contextmanager
+def mock_perm(perm='edx_proctoring.can_take_proctored_exam'):
```

Comment:
@MichaelRoytman I think this approach satisfies your concern in a relatively simple way.